### PR TITLE
Make path to plugins configurable in new pluginsystem

### PR DESCRIPTION
**Pull request by shyim**
## Description
Please describe your pull request:
* Why is it necessary?
Unify with "old" pluginsystem
* What does it improve?
Made plugin folder configureable
* Does it have side effects?
Nope

| Questions        | Answers
| ---------------- | -------------------------------------------------------
| BC breaks?       | no
| Tests pass?      | yes
| Related tickets? | 
| How to test?     | Change the folder in the config.php and try something with your plugins :)

### CODE REVIEW

**Comment by shyim** [comment]
@janbuecker travis need restart :3

**Comment by janbuecker** [comment]
Seems to be broken on travis. Bamboo's last build was successful. I've have a look into this.

File: engine/Shopware/Configs/Default.php [hunk 1]
```diff
@@ -96,9 +96,10 @@
         'apiEndpoint' => 'https://api.shopware.com',
     ],
     'plugin_directories' => [
-        'Default'   => $this->AppPath('Plugins_' . 'Default'),
-        'Local'     => $this->AppPath('Plugins_' . 'Local'),
-        'Community' => $this->AppPath('Plugins_' . 'Community'),
+        'Default'            => $this->AppPath('Plugins_' . 'Default'),
+        'Local'              => $this->AppPath('Plugins_' . 'Local'),
+        'Community'          => $this->AppPath('Plugins_' . 'Community'),
+        'ShopwarePlugins'    => dirname(dirname(dirname(__DIR__))) . '/custom/plugins'
```
Comment (FiveDigital) [inline engine/Shopware/Configs/Default.php]: Looks wrong

Comment (shyim) [reply engine/Shopware/Configs/Default.php]: I have replaced it with DocPath i guess it was depracted but it wasn't :)

**Pull request closed without merging**